src/audio/module_adapter/module/dolby/dax.c

```diff
@@ -48,6 +48,32 @@ static const char *get_params_str(const void *val, uint32_t val_sz)
 	return &params_str[0];
 }
 
+/**
+ * Same with cir_buf_copy in src/audio/component.c. We duplicate it here because cir_buf_copy
+ * is not an EXPORT_SYMBOL interface and cannot be accessed when dax is loaded as an llext module.
```

Comment:
is there anything that speaks against just exporting that function? Just add a commit to this series to do that (as the first one)